commands.txt

```diff
@@ -0,0 +1,82 @@
+## Commands
+This section lists command(s) run by bwamem2 workflow
+
+
+Split the fastq files into chunks to parallelize the alignment (optional).  If requested, subsequent steps will be run on each fastq chunk
+```
+
+        if [ -z "~{numReads}" ]; then
```

Comment:
You may want to edit this as the commands are not very readable to an average (non-developer) user as-is. I assume they are unedited snippets of code pulled from the workflow?

Reply:
updated